.gitignore

```diff
@@ -34,15 +34,21 @@ db/*.sqlite3
 *.rvmrc*
 *.sw?
 .sass-cache/**/*
-public/clicktale
 spec/reports
 token.txt
 /.idea
 /.env
 /.ssh
-/public
 /gems
 /.local
 /.wget-hsts
 .bash_history
 .byebug_history
+
+!public/
+
+public/*
+!public/static/
+
+public/static/*
+!public/static/bookplates/
```

Comment:
So the `public/static` directory is an artifact from the front-end build.  It's not necessarily required or encouraged, though if you want to, you're welcome to use it.

I'll have to make changes in Kubernetes for this, though.  Right now `/app/public` is an `emptyDir` volume so that the front-end can be built with an `initContainer`.

Reply:
Ok,I'm going to move it into `public/bookplates` to keep it separate from the front end build process. (Also one less line in .gitignore)